configs/sx8m_config

```diff
@@ -5103,17 +5103,17 @@ CONFIG_JFFS2_RTIME=y
 # CONFIG_JFFS2_RUBIN is not set
 # CONFIG_UBIFS_FS is not set
 # CONFIG_CRAMFS is not set
-CONFIG_SQUASHFS=m
+CONFIG_SQUASHFS=y
```

Comment:
Why moving from module to included in the kernel?

Reply:
So we don't have to load the module from a script in the initramfs environment when we mount an update image, and it does not significantly increase the kernel image size

Reply:
I don't agree with this approach. We should keep the kernel as slim as possible and load everything we can as a module, specially some code that we only use during updates. Otherwise we would have to move all the modules to within the kernel.
This is exactly the concept of modules: just attach to the kernel what you need when you need.

Reply:
Making it a module would require changes in the update system, it is a lot easier to leave it in.